cypress/integration/page.spec.js

```diff
@@ -1078,16 +1162,22 @@
 
       postDetails.comments().should('have.length', 6);
 
-      postDetails.comments().eq(5).byDataCy('CommentAuthor')
+      postDetails
+        .comments()
+        .eq(5)
+        .byDataCy('CommentAuthor')
         .should('have.text', 'Some name')
         .and('have.attr', 'href', 'mailto:some@email.com');
 
-      postDetails.comments().eq(5).byDataCy('CommentBody')
+      postDetails
+        .comments()
+        .eq(5)
+        .byDataCy('CommentBody')
         .should('have.text', 'Some comment body');
     });
 
     it('should show submit button spinner while waiting for server response', () => {
-      cy.clock()
+      cy.clock();
```

Comment:
There is a missing semicolon at the end of the `cy.clock` statement. Ensure that each statement is properly terminated to avoid syntax errors.